opensearch_domain.tf

```diff
@@ -46,15 +46,16 @@ resource "aws_opensearch_domain" "default" {
   }
 
   cluster_config {
-    instance_count           = var.instance_count
-    instance_type            = var.instance_type
-    dedicated_master_enabled = var.dedicated_master_enabled
-    dedicated_master_count   = var.dedicated_master_count
-    dedicated_master_type    = var.dedicated_master_type
-    zone_awareness_enabled   = var.zone_awareness_enabled
-    warm_enabled             = var.warm_enabled
-    warm_count               = var.warm_enabled ? var.warm_count : null
-    warm_type                = var.warm_enabled ? var.warm_type : null
+    instance_count                = var.instance_count
+    instance_type                 = var.instance_type
+    dedicated_master_enabled      = var.dedicated_master_enabled
+    dedicated_master_count        = var.dedicated_master_count
+    dedicated_master_type         = var.dedicated_master_type
+    multi_az_with_standby_enabled = var.multi_az_with_standby_enabled
```

Comment:
What version of the AWS provider was `multi_az_with_standby_enabled` added in? Does our current pin of the AWS provider in `versions.tf` require that version or above? Please look into this, update if needed, and then rerun the README generation with `make init && make readme`. Thanks!

Reply:
Thanks for the callout! It does need a minimum bump, the variable was introduced in https://github.com/hashicorp/terraform-provider-aws/blob/main/CHANGELOG.md#5150-august-31-2023. I've updated that value and run README generation. Note that that `make readme` fixes some things I missed when I did it manually, but also introduces some minor, seemingly unrelated changes (e.g., extra `/`).